metis/bin/metis_client

```diff
@@ -743,13 +743,11 @@ EOT
 
           if current_attempt_number <= max_number_attempts
             puts "UploadError -- retrying the upload"
-            MetisShell.log_error(e)
```

Comment:
Can you please revert this to include the `MetisShell.log_error(e)` calls, as in the main branch? We're using this method to provide better error messages to folks uploading data, so we don't want to lose it :smiley:

Reply:
🤦 the one time I didn't scroll all the way through the files-changed list...

Good catch.  Reverted.

This commit, https://github.com/mountetna/monoetna/commit/6698472b6664035341debaa1fbc3dfd645381666,  had appeared in a different way than I'm used to, and with attribution to me. So I'd reverted it thinking it was a problem, but clearly it isn't lol so, "*re-reverted" to add these changes back in!

Reply:
No worries! Thanks, this looks great!